src/helpers/helpers.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+let number = 0;
+
+const unicId = () => {
+  number++;
+
+  return number;

```

Comment:
better return id in type string and then you dont need to control type in all places